lms/djangoapps/verify_student/views.py

```diff
@@ -907,8 +907,14 @@ def _compose_message_reverification_email(
         context["verification_open"] = verification_open
         context["due_date"] = get_default_time_display(reverification_block.due)
         context["is_secure"] = is_secure
-        context["site"] = microsite.get_value('SITE_NAME', 'localhost')
-        context['platform_name'] = microsite.get_value('platform_name', settings.PLATFORM_NAME),
+
+        context['platform_name'] = microsite.get_value('platform_name', settings.PLATFORM_NAME)
+        context['support_link'] = microsite.get_value('email_from_address', settings.CONTACT_EMAIL)
+        context["allowed_attempts"] = allowed_attempts
+        context["used_attempts"] = used_attempts
+
+        site = microsite.get_value('SITE_NAME', 'localhost')
```

Comment:
Shouldn't this be: `microsite.get_value('SITE_NAME', settings.SITE_NAME)`?  (Otherwise this will try to use 'localhost' in production, right?)